.github/workflows/release.yml

```
@@ -0,0 +1,166 @@
+      - name: Package binaries
+        run: |
+          TAG="${GITHUB_REF_NAME:-${{ github.event.inputs.tag }}}"
+          STAGING="rgaa-${{ matrix.arch }}"
+          mkdir -p "${STAGING}"
+
+          cp "rgaa-rs/target/${{ matrix.target }}/release/rgaa-mcp" "${STAGING}/"
+          cp "rgaa-rs/target/${{ matrix.target }}/release/rgaa-cli" "${STAGING}/"
+
+          # obscura is Linux-only; macOS users must provide their own
+          cat > "${STAGING}/README" <<'README_EOF'
+rgaa-rs for macOS
```

actionlint (1.7.12)

[error] could not parse as YAML: could not find expected ':'

(syntax-check)